models/sun_2/model.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+
+<model>
+  <name>Sun 2</name>
+  <version>1.0</version>
+  <sdf version="1.5">model.sdf</sdf>
+
+  <author>
+    <name>Benjamin Perseghetti</name>
+    <email>bperseghetti@rudislabs.com</email>
+  </author>
+
+  <description>
+    A directional light for the sun with better default values.
+  </description>
+</model>
```

Comment:
We already have a sun model in this repo - can we replace it instead of adding a new one?

Reply:
This allows essentially for a sun with large shadows and one with minor/small shadows.

Reply:
If this is the case, can we specify this in the description or use more descriptive names?, I only realized this when you told me.

Reply:
I'm just going to remove the sun_2 model and do it the right way by also including the optional sunUCT, as well as sunSolarNoon,  sunHighShadow, and sunNone. I'll push that up here in a few after testing the new dynamic sun more.

Reply:
Done, removed and now have awesome properly generated sun.